models/spleen_deepedit_annotation/configs/multi_gpu_train.json

```diff
@@ -31,6 +31,7 @@
         "$monai.utils.set_determinism(seed=123)"
     ],
     "run": [
+        "$@validate#handlers#0.set_trainer(trainer=@train#trainer) if @early_stop else None",
```

Comment:
Hi @wyli @Nic-Ma , do you know if we can use early stopping with distributed training？
I tried with this config but the program hangs in `finalize` (when execute `dist.destroy_process_group()`).

Reply:
fyi https://github.com/pytorch/ignite/pull/666

Reply:
perhaps we can enhance the docstring here for future references https://github.com/Project-MONAI/MONAI/blob/6dfe17a7bfda9b4ddfa19b60886b7fa14e816afd/monai/handlers/earlystop_handler.py#L50-L51